app/src/main/res/layout/dialog_image_description.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingBottom="0dp">
+
+    <com.github.chrisbanes.photoview.PhotoView
+        android:id="@+id/imageDescriptionView"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        android:contentDescription="@string/post_media_image"/>
+
+    <com.google.android.material.textfield.TextInputLayout
+        style="@style/TuskyTextInput"
+        android:id="@+id/imageDescriptionTextLayout"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        android:layout_marginStart="?dialogPreferredPadding"
+        android:layout_marginTop="?dialogPreferredPadding"
+        android:layout_marginEnd="?dialogPreferredPadding"
+        app:hintEnabled="false"
+        app:counterEnabled="true"
+        app:counterTextColor="?android:textColorTertiary">
```

Comment:
Looks like the counter is hidden because the TextInputLayout stretches too high.

Also, it probably makes sense to set the `counterMaxLength`.

Reply:
I ignored that for the moment:

- 1.500 is a rather high limit (to reach)
- Showing the limit (below) would take some vertical space
- The text field _will_ adhere to the limit and stop accepting input (which is some sort of queue in the right direction of what's happening)

Reply:
ok, but then please remove the counter if it isn't shown anyway

Reply:
Done.